lib/statsample/test/bartlettsphericity.rb

```diff
@@ -31,7 +31,7 @@ def initialize(matrix,ncases)
       # 
       def compute
         @value=-((@ncases-1)-(2*@nvars+5).quo(6))*Math::log(@matrix.determinant)
-        @df=(@nvars*(@nvars-1)).quo(2)
+        @df=(@nvars*(@nvars-1)) / 2
```

Comment:
I did this because the `distribution` gem previously used only GSL for computation, and `@df`, when it evaulated to a `Rational` with `quo` would get accepted by GSL. This will prevent generation on rational numbers.